model/src/main/resources/reference.yml

```diff
@@ -236,8 +238,10 @@ ort:
         clientPassword: clientPassword
         token: token
 
+    # Storage readers are listed from highest to lower priority, i.e. the first match wins.
     storageReaders: [local, postgres, http, clearlyDefined]
 
+    # For storage writers no priority is implied by the order; all writers are written to.
```

Comment:
Nit: Maybe "all writers are invoked" or "data is stored in all writers" ?

Reply:
I've reworded yet slightly differently.